dyn_em/module_initialize_real.F

```diff
@@ -665,6 +665,16 @@ SUBROUTINE init_domain_rk ( grid &
          !  coarse grid since we are going to make the interface consistent
          !  in the model betwixt the CG and FG domains.
 
+         !  An important point is to inform the user if their request cannot
+         !  be satisfied. Do not skip over this quietly.
+
+         IF ( ( config_flags%smooth_cg_topo ) .AND. &
+              ( internal_time_loop .EQ. 1 ) .AND. &
+              ( grid%id .EQ. 1 ) .AND. &
+              ( flag_soilhgt .NE. 1) ) THEN
+            CALL wrf_error_fatal(' --- ERROR: NML option smooth_cg_topo=T, but found no soil elevation data in metgrid files')
```

Comment:
@davegill 
Could you add something like "If using ERA5 input, must use include all invariant files."